ddtrace/opentracer/tracer.py

```diff
@@ -274,7 +297,7 @@ def start_span(
         # set the start time if one is specified
         ddspan.start = start_time or ddspan.start
 
-        otspan = Span(self, ot_parent_context, operation_name)
+        otspan = Span(self, ot_parent_context, operation_name)  # type: ignore[arg-type]
```

Comment:
which are we ignoring? would it work better as multi-line so you can ignore only the part you need?

Reply:
(this assumes it is a Span() argument not the otspan... or else we could cast?

Reply:
Same problem with the operation name being an optional in `start_span()` but required in the actual Span constructor.